### PR TITLE
Update IRC references to (live) chat

### PR DESCRIPTION
**Pull request by reosarevok**
We've moved to use Matrix as our main chat platform, so this changes references to IRC to point to our new generic page.

### CODE REVIEW

File: root/account/Register.js [hunk 1]
```diff
@@ -78,11 +78,11 @@ component Register(
...
-           To talk to other users, try the {fo|forums} or {irc|IRC}.`,
+           To talk to other users, try the {fo|forums} or the {chat|chat}.`,
           {
             bl: 'http://blog.metabrainz.org/',
             fo: 'https://community.metabrainz.org/',
-            irc: '/doc/Communication/IRC',
+            irc: '/doc/Communication/ChatBrainz',
             tw: 'https://twitter.com/MusicBrainz',
           },
         )}
```
Comment (yvanzo) [inline root/account/Register.js:88]: I didn’t test but I expect that the variable `irc` should be updated too.

I suggest to use longer variable names such as `blog_link`, `forum_link`, `chat_link`, `twitterx_link` to make these self-explanatory and more easily identifiable as variable names by translators.

Also, maybe _live chat_ can be used for the link title as in other files.

**Review by yvanzo — CHANGES_REQUESTED** [review]

File: root/layout/components/Footer.js [hunk 2]
```diff
@@ -26,8 +26,8 @@ component Footer() {
         <a className="internal" href={DONATE_URL}>{l('Donate')}</a>
         <a className="internal" href="//wiki.musicbrainz.org/">{l('Wiki')}</a>
         <a className="internal" href="https://community.metabrainz.org/">{l('Forums')}</a>
-        <a className="internal" href="/doc/Communication/IRC">
-          {l('Chat (IRC)')}
+        <a className="internal" href="/doc/Communication/ChatBrainz">
+          {l('Live chat')}
```
Comment (mwiencek) [inline root/layout/components/Footer.js:30]: Live chat kinda makes me think of those customer service popups on every website. 😅

File: root/account/Register.js [hunk 3]
```diff
@@ -78,11 +78,11 @@ component Register(
       <p>
         {exp.l(
           `Follow our {bl|blog} or {tw|twitter account}!
-           To talk to other users, try the {fo|forums} or {irc|IRC}.`,
+           To talk to other users, try the {fo|forums} or the {chat|chat}.`,
```
Comment (mwiencek) [inline root/account/Register.js:81]: Shouldn't this also say live chat? Or can we just use "chat" everywhere?

Comment (yvanzo) [reply root/account/Register.js:81]: I don’t have any argument for one or the other.

File: root/layout/components/Footer.js (hunk 2 above)
Comment (reosarevok) [reply root/layout/components/Footer.js:30]: I know - maybe just "chat" is good enough? I wasn't sure what to call it, I wanted to make it clear it's a place to talk synchronously as opposed to the forum.

Comment (mwiencek) [reply root/layout/components/Footer.js:30]: "Chat" seems fine. My other thought was "real-time chat," which might sound too technical (though not nearly as much as "IRC" :)).

File: root/account/Register.js (hunk 1 above)
Comment (yvanzo) [reply root/account/Register.js]: > I suggest to use longer variable names such as `blog_link`, `forum_link`, `chat_link`, `twitterx_link` to make these self-explanatory and more easily identifiable as variable names by translators.

This part has been agreed-upon in the chat, and is implemented through commit https://github.com/metabrainz/musicbrainz-server/pull/3287/commits/97793f745f6c2285724cfad20492a2b21c295c30.

**Review by yvanzo — APPROVED** [review]
:ship:

**Pull request merged**